latest/index.bs

```diff
@@ -331,24 +317,1002 @@ Conforming readers:
 - MAY ignore other groups or arrays under the root of the hierarchy.
 
 
+"coordinateSystems" metadata {#coord-sys-md}
+--------------------------
+
+A "coordinate system" is a collection of "axes" / dimensions with a name. Every coordinate system:
```

Comment:
maybe simpler to say 'A "coordinate system" is a collection of "axes"...', i.e. omit the term "dimensions", since it's not really defined, unlike "axes"